20.md

```diff
@@ -1,93 +1,9 @@
 NIP-20
 ======
 
-
 Command Results
 ---------------
 
-`draft` `optional` `author:jb55`
-
-When submitting events to relays, clients currently have no way to know if an event was successfully committed to the database. This NIP introduces the concept of command results which are like NOTICE's except provide more information about if an event was accepted or rejected.
-
-A command result is a JSON object with the following structure that is returned when an event is successfully saved to the database or rejected:
-
-    ["OK", <event_id>, <true|false>, <message>]
-
-Relays MUST return `true` when the event is a duplicate and has already been saved. The `message` SHOULD start with `duplicate:` in this case.
-
-Relays MUST return `false` when the event was rejected and not saved.
-
-The `message` SHOULD provide additional information as to why the command succeeded or failed.
-
-The `message` SHOULD start with `blocked:` if the pubkey or network address has been blocked, banned, or is not on a whitelist.
-
-The `message` SHOULD start with `invalid:` if the event is invalid or doesn't meet some specific criteria (created_at is too far off, id is wrong, signature is wrong, etc)
-
-The `message` SHOULD start with `pow:` if the event doesn't meet some proof-of-work difficulty. The client MAY consult the relay metadata at this point to retrieve the required posting difficulty.
-
-The `message` SHOULD start with `rate-limited:` if the event was rejected due to rate limiting techniques.
-
-The `message` SHOULD start with `error:` if the event failed to save due to a server issue.
-
-Ephemeral events are not acknowledged with OK responses, unless there is a failure.
```

Comment:
Is this not a thing anymore? It would be rather good because it seems to be good to actually acknowledge NIP-42 AUTH event

Reply:
I didn't remember this, or maybe never realized it was there. What was the reasoning?

Reply:
I don't know but I would expect something like avoiding traffic generated by events like typing... indicator. Good if we remove this line